politeiawww/www.go

```diff
@@ -57,9 +59,45 @@ const (
 	csrfKeyLength = 32
 )
 
+func convertWWWErrorStatusFromPD(e pd.ErrorStatusT) www.ErrorStatusT {
+	switch e {
+	case pd.ErrorStatusInvalidRequestPayload:
+		// Intentionally omitted because this indicates a politeiawww
+		// server error so a ErrorStatusInvalid should be returned.
+	case pd.ErrorStatusInvalidChallenge:
+		// Intentionally omitted because this indicates a politeiawww
+		// server error so a ErrorStatusInvalid should be returned.
+	case pd.ErrorStatusInvalidFilename:
+		return www.ErrorStatusInvalidFilename
+	case pd.ErrorStatusInvalidFileDigest:
+		return www.ErrorStatusInvalidFileDigest
+	case pd.ErrorStatusInvalidBase64:
+		return www.ErrorStatusInvalidBase64
+	case pd.ErrorStatusInvalidMIMEType:
+		return www.ErrorStatusInvalidMIMEType
+	case pd.ErrorStatusUnsupportedMIMEType:
+		return www.ErrorStatusUnsupportedMIMEType
+	case pd.ErrorStatusInvalidRecordStatusTransition:
+		return www.ErrorStatusInvalidPropStatusTransition
+	}
+	return www.ErrorStatusInvalid
+}
+
+func convertWWWErrorStatus(pluginID string, errCode int) www.ErrorStatusT {
+	switch pluginID {
+	case "":
+		// politeiad API
+		e := pd.ErrorStatusT(errCode)
+		return convertWWWErrorStatusFromPD(e)
+	}
+
+	// No corresponding www error status found
+	return www.ErrorStatusInvalid
+}
+
 // Fetch remote identity
-func (p *politeiawww) getIdentity() error {
```

Comment:
Lot's of files that need to be eliminated.

Reply:
I think i've cleaned it up as much as possible.  Once cms migrates we can delete a bunch of stuff.